global.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is this file? If it's not directly related to the project, let's add it to gitignore or just remove it from this PR.

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)